test/lint/newline-after-block.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Checking that functional blocks are followed by newlines..."
 
-files=$(find ./ -type f -name "*.go")
+files=$(git ls-files "*.go")
```

Comment:
AFAIK this doesn't include uncommitted files, if that is the case, then I don't think we want this change as running `make static-analysis` should work on uncommitted files.